docs/sections/getting_started.md

```diff
@@ -128,17 +128,23 @@ popper run
 
 You should see the output of actions printed to the terminal.
 
+To better understand what the above code does, we include a help flag:
```

Comment:
We can say "To obtain more detailed information about what the command does, you can pass the `--help` flag:`

And as it was mentioned below, we can add the note that all subcommands take a `--help` flag.

Reply:
Alright, thanks I'd effect this change.